test/Sentry.Tests/SentrySdkTests.cs

```diff
@@ -79,22 +79,6 @@ public void Init_ValidDsn_EnablesSdk()
         Assert.True(SentrySdk.IsEnabled);
     }
 
-    [Fact]
-    public void Init_ValidDsnWithSecret_EnablesSdk()
```

Comment:
This makes sure we don't regress on old DSNs. Which could still be out there

Reply:
I was conflicted about this too, but it should go at some point, so...